frontend/src/components/ProductView/ProductIngredients.tsx

```
@@ -0,0 +1,73 @@
+  return (
+    <div className="flex flex-col items-start gap-6 w-full">
+      <h2 className="text-[22px] font-medium leading-7 text-black">{t('product.keyIngredients')}</h2>
+
+      <div className="w-full overflow-hidden">
+        <div
+          className="flex gap-6 cursor-pointer"
+          style={{
+            animation: isPaused ? 'none' : `scroll ${ingredients.length * 8}s linear infinite`,
+          }}
+          onClick={togglePause}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)